src/api.ts

```diff
@@ -391,6 +392,7 @@ export class OpenSeaAPI {
       ...opts,
       headers: {
         ...(apiKey ? { "X-API-KEY": apiKey } : {}),
+        ...{ "sdk-version": sdkVersion },
```

Comment:
```suggestion
        ...{ "SDK-Version": sdkVersion },
```

Just to match the usual header format